index.bs

```diff
@@ -863,7 +863,7 @@ When this method is invoked, the user agent MUST run the following steps:
 
 <div class="algorithm" data-algorithm="run-animation-frames">
 
-When an {{XRSession}} |session| receives updated [=viewer=] state from the [=XRSession/XR device=], it runs an <dfn>XR animation frame</dfn> with a timestamp |now| and an {{XRFrame}} |frame|, which MUST run the following steps regardless of if the [=list of animation frame callbacks=] is empty or not:
+When an {{XRSession}} |session| receives updated [=viewer=] state from the [=XRSession/XR device=], it runs an <dfn>XR animation frame</dfn> with a timestamp |now| and an {{XRFrame}} |frame| with [=XRFrame/time=] |now|, which MUST run the following steps regardless of if the [=list of animation frame callbacks=] is empty or not:
```

Comment:
This doesn't feel like the right way to specify this. `|now|` and `|frame|` are both inputs to the algorithm, but the new text feels like an assignment happening in the algorithm description.

I'd suggest either:
 - Assigning `[=XRFrame/time=]` to `|now|` explicitly in the algorithm or
 - Removing |now| from the algorithm description altogether and using [=XRFrame/time=] in it's place elsewhere in the algorithm.

Reply:
Done

Reply:
Hm... I think we went to far in the other direction. We still need to have a declaration of the `XRFrame` somewhere.

Reply:
Not sure what you mean by that, it's constructed in the algorithm itself (as it should be, IMO, previously it just appeared out of the ether)

Reply:
Oh, my bad, i pushed up a partial commit because I forgot to save

Reply:
🤣